Index: s3cmd-0.9.4/setup.py
===================================================================
--- s3cmd-0.9.4.orig/setup.py	2007-08-24 23:40:39.000000000 +0700
+++ s3cmd-0.9.4/setup.py	2007-08-24 23:40:53.000000000 +0700
@@ -9,13 +9,13 @@
 	sys.stderr.write("S3cmd required Python 2.4 or newer.\n")
 	sys.exit(1)
 
-try:
-	import elementtree.ElementTree as ET
-except ImportError, e:
-	sys.stderr.write(e.message + "\n")
-	sys.stderr.write("Please upgrade to Python 2.5 or install ElementTree module\n")
-	sys.stderr.write("from http://effbot.org/zone/element-index.htm\n")
-	sys.exit(1)
+#try:
+#	import elementtree.ElementTree as ET
+#except ImportError, e:
+#	sys.stderr.write(e.message + "\n")
+#	sys.stderr.write("Please upgrade to Python 2.5 or install ElementTree module\n")
+#	sys.stderr.write("from http://effbot.org/zone/element-index.htm\n")
+#	sys.exit(1)
 
 try:
 	## Remove 'MANIFEST' file to force
